assets/pages/bundles-landing/bundlesLanding.jsx

```diff
@@ -43,7 +43,7 @@ const content = (
       <Bundles />
       <WhySupport />
       <WaysOfSupport />
-      <LinksFooter />
+      <Footer privacyPolicy />
```

Comment:
Where is `privacyPolicy` defined? I can't see it in the imports :-/

Reply:
It's not, that's a boolean.

Reply:
Is it possible to do something more readable like `privacyPolicy = true`?

Reply:
That causes a lint error, it tells you to drop the `true`.